### ISSUE TITLE
Provide design file input

### ISSUE THREAD

**Issue by apeltzer** [nf-core/hlatyping#104]
Having some type of `--input` parameter and thus a sample sheet would be very nice 👍🏻

**Comment by szymonwieloch** [comment]
Hi! I also had some issues because of missing input sheet file and maybe I can describe them to you, so that you can make up your mind about this feature.

1. The current --input <glob regex> approach does not work in certain scenarios. I have a directory structure where the directories (not files) are symlinks. It turns out that the NextFlow `file(<glob regex>)` function does not handle those cases correctly. You need to specify all files manually.
2. I also had a long list of directories with FASTQ files. I wanted to process only a subset of them. This was extremely difficult because the glob regex `/some/path/*/*.fastq.gz` matched all files and there is no easy way to select concrete directories.
3. input_paths parameter did work for me and solved the problem. However it is not documented. At first I missed it and had to manually change my environment to make the pipeline run.

To sum up, it would be really nice to have an input TSV file with the following example format:

```tsv
ID    R1    R2
EXAMPLE_ID    /example/path/data_R1.fastq.gz     /example/path/data_R2.fastq.gz
```

Of course this would have to be documented so that users know about such a feature. NextFlow handles TSV format easily with the `splitCsv(header: true, sep:'\t')` operator.

**Comment by christopher-mohr** [comment]
Solved by #140.
